tests/test_build.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+import pytest
+from command.build import BuildCmd
+
+
+def test_create_bundle():
+    paths = ["tests/test_files/csv.yaml", "tests/test_files/crd.yaml", "tests/test_files/package.yaml"]
+    yamls = []
+    for x in range(3):
```

Comment:
```python
for path in paths:
    with open(path) as f:
        ...
```

Reply:
@MartinBasti addressed in latest commit.